packages/loot-core/src/server/db/mutations/transactions.ts

```
@@ -0,0 +1,14 @@
+import { insertWithSchema, updateWithSchema, delete_ } from "../db";
+import { DbTransaction } from "../types";
+
+export function insertTransaction(transaction: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)